frontend/model/contracts/shared/constants.js

```diff
@@ -51,6 +51,7 @@ export const CHATROOM_NAME_LIMITS_IN_CHARS = 50
 export const CHATROOM_DESCRIPTION_LIMITS_IN_CHARS = 280
 export const CHATROOM_ACTIONS_PER_PAGE = 40
 export const CHATROOM_MAX_ARCHIVE_ACTION_PAGES = 2 // 2 pages of actions
+export const CHATROOM_POLL_DURATION_DAYS = 7 // TODO: discuss with team and confirm this value
```

Comment:
For now, a poll expires 7 days after its creation. '7' here is just a random number I chose for development and let me know what this should be. will tweak this value.

Reply:
Most apps allow the user to specify the duration. If this wasn't in the designs... well, maybe add a dropdown where the user can select how many days the post expires in, from 1 day to 30 days. Can be done as part of a separate PR too, based on what you think looks good, or after @leihla designs it. I'll leave the decision up to you @SebinSong.

Reply:
Added a comment for @leihla on Figma regarding this: https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN?node-id=23766:265445#468454194

Reply:
@taoeffect Sure, I would love to work after @leihla 's design update on this.

Reply:
> For now, a poll expires 7 days after its creation.

The current design on Figma specifies that polls close after 2 months by default, so this should be `60` instead of `7`. (The note is on a later frame, here: https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN/Group-Income-2.0?type=design&node-id=5159-0&t=B3OUjrU5YYYuhjZT-0)

Reply:
updated the PR [here](https://github.com/okTurtles/group-income/pull/1628/commits/284c83fd6d9da5e88c27fef708f599c26e7431df) per @dotmacro 's comment